lib/blockscore/question_set.rb

```diff
@@ -0,0 +1,31 @@
+require 'blockscore/actions/create'
+require 'blockscore/actions/retrieve'
+require 'blockscore/actions/all'
+
+module BlockScore
+  class QuestionSet < Base
+    include BlockScore::Actions::Create
+    include BlockScore::Actions::Retrieve
+    include BlockScore::Actions::All
+
+    def create(params)
+      self.class.create(params)
+    end
+
+    def retrieve(id)
+      self.class.retrieve(id)
+    end
+
+    def all(options = {})
+      self.class.all(options)
+    end
+    
+    def score(answers)
+      self.class.post "#{self.class.endpoint}#{id}/score", :answers => answers
+    end
+
+    def create
+      self.class.create(:person_id => person_id)
+    end
+  end
+end
```

Comment:
If we are only supporting 1.9.3 and up then this file is a prime candidate for [`Forwardable`](http://ruby-doc.org/stdlib-1.9.3/libdoc/forwardable/rdoc/Forwardable.html)